data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
> If something else comes up, we can add other CSS files and load them conditionally based on version, like is done for 3.20.

Mmyeah indeed, fair point.
Although it does make it a bit more complex adding specifics; before you could `@import` e.g. 3.0 into 3.18 but not into 3.20, and decide that in the 3.18-specific file, not inside the code.